.travis.yml

```diff
@@ -34,4 +31,5 @@ script:
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_snmp_information
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_users
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_traceroute
+- nosetests -v TestNXOSDriver:TestNXOSDriver_bgp_neighbors
```

Comment:
I don't mind custom tests but three things:

1. As it's a set of getters include them inside `TestGetterNXOSDriver`.
2. Rename the methods to `test_nxos_only_something_something` to make it clear it's a test defined only for nxos.
3. Enable the global test as well: `nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_bgp_neighbors`.

I really like what you did here with the tests. I need to find the time to redo the testing framework to support different test cases per test.